### ISSUE TITLE
Additional self.loss in line 121

### ISSUE THREAD

**Issue by DerekChia** [nathanrooy/word2vec-from-scratch-with-python#1]
Hello, I see an additional line of self.loss in your implementation in line 121.

Is this intended?

`self.loss += -2*np.log(len(w_c)) -np.sum([u[word.index(1)] for word in w_c]) + (len(w_c) * np.log(np.sum(np.exp(u))))`

Thanks.

**Comment by nathanrooy** [comment]
Thanks, I've commented out the extra one.
